.storybook/utils.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  array,
+  boolean,
+  button,
+  color,
+  date,
+  files,
+  number,
+  object,
+  optionsKnob as options,
+  radios,
+  select,
+  text
+} from "@storybook/addon-knobs";
+
+export const darkBackground = [
+  {
+    name: "Dark",
+    value: "#202020",
```

Comment:
nitpick: maybe we could import this color from calcite colors?

Reply:
💥